docs/Time_in_Arbitrum.md

```diff
@@ -4,52 +4,55 @@ title: Time in Arbitrum
 sidebar_label: Time in Arbitrum
 ---
 
-As on Ethereum, Arbitrum doesn't use real wall-clock time, but instead uses block numbers, which correspond to block numbers on the main Ethereum chain.
+Arbitrum supports both block numbers which correspond to block numbers on the main Ethereum chain as well as an approximation of the current timestamp,
 
 Also as on Ethereum, clients submit transactions they want to see happen, and the system (usually) executes those transactions at some later time.
-In Arbitrum Rollup, clients submit transactions by posting messages to the Ethereum chain.
+In Arbitrum Rollup, clients submit transactions by posting messages to the Ethereum chain, either directly or through an aggregator.
 These messages are put into the chain's _inbox_.
 Every message is timestamped with the time it was put into the inbox.
 
 Messages in an ArbChain's inbox are processed in order.
 Generally, some time will elapse between the time when a message is put into the inbox (and timestamped) and the time when the contract processes the message and carries out the transaction requested by the message.
 
-If your Solidity contract, running on Arbitrum Rollup, accesses `block.number`, this will give you the ArbChain's current time.
-Because ArbChains operate off-chain, an ArbChain's time might differ a little bit from the current Ethereum block time.
-The ArbChain's clock will never get ahead of the Ethereum block time, but the ArbChain's clock might "run a bit slow".
-In particular, it might be as many as 20 blocks behind the true Ethereum time.
+If your Solidity contract, running on Arbitrum Rollup, accesses `block.number` or `block.timestamp`, this will give you the ArbChain's current block or timestamp respectively.
+Because ArbChains operate off-chain, an ArbChain's block number or timestamp might differ a little bit from the current Ethereum block.
+The ArbChain's clock will never get ahead of the Ethereum's, but the ArbChain's clock might "run a bit slow".
+A rollup chain can configure the maximum amount its clock can differ, but generally it will stay within 20 blocks or 10 minutes
+In particular, a rollup chain can configure how far behind it's clock may fall, but generally it will stay within 20 blocks or 10 minutes.
```

Comment:
Typo:  should be "its" instead of "it's".

Reply:
Fixed